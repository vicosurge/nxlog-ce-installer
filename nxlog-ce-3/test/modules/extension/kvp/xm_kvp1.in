key1:value1 key2:value2
key1:'value1' key2:'value2'
'key1':value1 'key2':value2
'key1':'value1' 'key2':'value2'
key1=value1 key2=value2
key1=value1,key2=value2
key1=value1;key2=value2
key1='value 1';key2='value 2'
'key 1'='value 1';'key 2'='value 2'
key1=value1, key2=value2
key1=value1; key2=value2
key1: value1; key2: value2
'key 1': value1; 'key 2': value2
'key 1': 'value 1'; 'key 2': 'value 2'
'key 1': 'value 1 '; 'key 2': 'value 2 '
'key 1': ' value 1 '; 'key 2': ' value 2 '
key1:value1  key2:value2
key1:value1;;key2:value2
key1=value1, key2='value 2', key3=value-3
key1:value1 key2:value2 key3:value3
key1=value1;key2=value2;key3=value3
key1=value1; key2=value2;
key1=value1;key2=value2;key3=value3;
key1=value1; key2=value2; key3=value3;
