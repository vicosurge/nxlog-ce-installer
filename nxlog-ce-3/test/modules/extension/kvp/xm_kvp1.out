{"SourceModuleName":"in","SourceModuleType":"im_file","key1":"value1","key2":"value2"}
{"SourceModuleName":"in","SourceModuleType":"im_file","key1":"value1","key2":"value2"}
{"SourceModuleName":"in","SourceModuleType":"im_file","key1":"value1","key2":"value2"}
{"SourceModuleName":"in","SourceModuleType":"im_file","key1":"value1","key2":"value2"}
{"SourceModuleName":"in","SourceModuleType":"im_file","key1":"value1","key2":"value2"}
{"SourceModuleName":"in","SourceModuleType":"im_file","key1":"value1","key2":"value2"}
{"SourceModuleName":"in","SourceModuleType":"im_file","key1":"value1","key2":"value2"}
{"SourceModuleName":"in","SourceModuleType":"im_file","key1":"value 1","key2":"value 2"}
{"SourceModuleName":"in","SourceModuleType":"im_file","key 1":"value 1","key 2":"value 2"}
{"SourceModuleName":"in","SourceModuleType":"im_file","key1":"value1","key2":"value2"}
{"SourceModuleName":"in","SourceModuleType":"im_file","key1":"value1","key2":"value2"}
{"SourceModuleName":"in","SourceModuleType":"im_file","key1":"value1","key2":"value2"}
{"SourceModuleName":"in","SourceModuleType":"im_file","key 1":"value1","key 2":"value2"}
{"SourceModuleName":"in","SourceModuleType":"im_file","key 1":"value 1","key 2":"value 2"}
{"SourceModuleName":"in","SourceModuleType":"im_file","key 1":"value 1 ","key 2":"value 2 "}
{"SourceModuleName":"in","SourceModuleType":"im_file","key 1":" value 1 ","key 2":" value 2 "}
{"SourceModuleName":"in","SourceModuleType":"im_file","key1":"value1","key2":"value2"}
{"SourceModuleName":"in","SourceModuleType":"im_file","key1":"value1","key2":"value2"}
{"SourceModuleName":"in","SourceModuleType":"im_file","key1":"value1","key2":"value 2","key3":"value-3"}
{"SourceModuleName":"in","SourceModuleType":"im_file","key1":"value1","key2":"value2","key3":"value3"}
{"SourceModuleName":"in","SourceModuleType":"im_file","key1":"value1","key2":"value2","key3":"value3"}
{"SourceModuleName":"in","SourceModuleType":"im_file","key1":"value1","key2":"value2"}
{"SourceModuleName":"in","SourceModuleType":"im_file","key1":"value1","key2":"value2","key3":"value3"}
{"SourceModuleName":"in","SourceModuleType":"im_file","key1":"value1","key2":"value2","key3":"value3"}
